SwrveSDK/src/main/java/com/swrve/sdk/messaging/SwrveMessage.java

```diff
@@ -19,7 +19,7 @@
  * In-app message inside a campaign, with different formats.
  */
 public class SwrveMessage {
-    protected static final String LOG_TAG = "SwrveSDK";
+    protected static final String LOG_TAG = "SwrveMessage";
```

Comment:
Same question as I asked above....should we use one standard LOG_TAG everywhere....."SwrveSDK"?